simulation/README.md

```diff
@@ -0,0 +1,48 @@
+# 🤖 Streaming Simulator
+A simulator for throughput, network use, and shuffle quality with MosaicML Streaming. The simulator allows you to:
+- Plan runs and anticipate issues beforehand
+- Find optimal run configurations
+- Debug issues with underperforming runs
+- Better understand the impact of different configurations
+
+## 🚀 Getting Started
+Run the following to install simulator-specific dependencies, if they don't already exist:
+```
+pip install --upgrade "mosaicml-streaming[simulator]"
+```
+Then, simply run `simulator` in your command line to open the Web UI and get simulating!
```

Comment:
Should we rename this to `sd_simulator` or `sd-simulator` or something else other than `simulator`? Wondering, if there is a name collision with other application or not?

Reply:
google search for a `simulator` command on linux and mac doesn't turn up any results...so we should probably be fine? I think the simpler the better right

Reply:
discussed and resolved. if issues, will change later!